pkg/operator/ceph/cluster/osd/status.go

```diff
@@ -235,6 +235,13 @@ func (c *Cluster) completeOSDsForAllNodes(config *provisionConfig, configOSDs bo
 				currentTimeoutMinutes++
 				if currentTimeoutMinutes == timeoutMinutes {
 					config.addError("timed out waiting for %d nodes: %+v", remainingNodes.Count(), remainingNodes)
+					//start to remove remainingNodes waiting timeout.
+					for remainingNode := range remainingNodes.Iter() {
+						clearNodeName := k8sutil.TruncateNodeName(orchestrationStatusMapName, remainingNode)
+						if err := c.kv.ClearStore(clearNodeName); err != nil {
+							config.addError("removing timeout node %+v error with node name %+v: %+v", remainingNode, clearNodeName, err)
```

Comment:
Just one more suggestion, how about this message?
```
config.addError("failed to clear node %s status with name %s. %+v", remainingNode, clearNodeName, err)
```

Reply:
It is more clear, sure I will modify the error message, Thank you, @travisn.